.github/workflows/build.yml

```diff
@@ -10,6 +10,9 @@ on:
   pull_request:
   schedule:
     - cron: "10 0,12 * * *"
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/master' && !startsWith(github.ref, 'refs/tags/v') }}
```

Comment:
Does this work for v1.x branches?

Reply:
Oooh, good point, that means I think it'll need to be something like:

```
${{ github.ref != 'refs/heads/master' && !startsWith(github.ref, 'refs/heads/v') && !startsWith(github.ref, 'refs/tags/v') }}
```

That should cover both branches and tags.

Reply:
Updated it. Thanks.